app/scripts/lib/ppom/ppom-util.ts

```diff
@@ -37,18 +39,37 @@ type PPOMRequest = Omit<JsonRpcRequest, 'method' | 'params'> & {
   method: typeof METHOD_SEND_TRANSACTION;
   params: [TransactionParams];
 };
+
 export async function validateRequestWithPPOM({
   ppomController,
   request,
   securityAlertId,
   chainId,
+  updateSecurityAlertResponse: updateSecurityResponse,
 }: {
   ppomController: PPOMController;
   request: JsonRpcRequest;
   securityAlertId: string;
   chainId: Hex;
+  updateSecurityAlertResponse: (
+    method: string,
+    securityAlertId: string,
+    securityAlertResponse: SecurityAlertResponse,
+  ) => void;
 }): Promise<SecurityAlertResponse> {
   try {
+    if (!(await isChainSupported(chainId))) {
+      return {
```

Comment:
We're returning this new response but should we also update so the controllers have the correct chain not supported response on them?

Reply:
That is true. I pushed a change to update it before responding.